xapian-applications/omega/Makefile.am

```diff
@@ -164,12 +166,16 @@ omindex_SOURCES = omindex.cc myhtmlparse.cc htmlparse.cc index_file.cc mime.cc\
  runfilter.cc freemem.cc common/msvc_dirent.cc xpsxmlparse.cc common/str.cc\
  pkglibbindir.cc svgparse.cc tmpdir.cc urlencode.cc atomparse.cc xlsxparse.cc\
  opendocparse.cc common/keyword.cc msxmlparse.cc common/safe.cc timegm.cc\
- datetime.cc
+ datetime.cc common/closefrom.cc worker.cc worker_comms.cc
 if NEED_MKDTEMP
 omindex_SOURCES += portability/mkdtemp.cc
 endif
 omindex_LDADD = $(MAGIC_LIBS) $(XAPIAN_LIBS) $(ZLIB_LIBS) libutf8convert.la
 
+omindex_pdf_SOURCES = assistant.cc worker_comms.cc handler_pdf.cc
+omindex_pdf_LDADD = -lpoppler-cpp $(XAPIAN_LIBS)
+omindex_pdf_LDFLAGS = -pthread
+
```

Comment:
Here I don't know how to add the -pthread flag in a proper way (e.g. add it just if it is necessary).